---
layout: page
title: Expression in a GRAS Organism
---

<div class="container">
	<div class="row row-centered">
		<div class="col-centered blue col-xs-12">
			<h1>Expression in a GRAS Organism</h1>
			<p>Our project focuses on how the GAF proteins can be used as a food colorant. However, in order to bring this protein into the human food system, we need protein expression to occur in a Generally Recognized as Safe, or GRAS organism. Since Escherichia coli has naturally occurring endotoxins, it is not given the GRAS designation. Bacillus subtilis on the other hand is considered GRAS and is widely used in the food industry. For these reasons we explored how we would bring this protein production into B. subtilis and while we did not have the opportunity to complete our experiments, we did establish a thorough experimental setup for the future. </p>
		</div>
	</div>
	<div class="row row-centered">
		<div class="col-centered col-xs-12">
<h3>Motivation </h3>
<p>The primary motivation of experimentation with <em>Escherichia coli</em> was the scope of characterization available for this organism and its familiarity with the iGEM community. <em>E. coli </em>has a plethora of knowledge base around the organism with many available parts, protocols, and transformation success. <em>E. coli</em> consists of a diverse group of bacteria where most are harmless and are an important part of a normal-healthy intestinal tract (BW.3). However, since <em>E. coli </em>has naturally occurring endotoxins, it is not suitable for human consumption. <em>Bacillus subtilis </em>on the other hand, is considered a GRAS (Generally Recognized as Safe) organism and is widely used in the food industry. This bacteria constitutes the main component of the soybean dish, Natto, and is widely used by established biotechnically companies like NovaZymes and DSM (18, 19). For these reason, in the future we would like to transfer the CBCR expression into <em>B. subtilis. </em></p>
<p><em>Bacillus</em> bacteria are among the most ubiquitous microorganisms present in nature and can be found in water, soil, and in air (as spores) [BW.4]. <em>The Bacillus</em> genus consists of Gram-positive, rod-shaped bacteria (BW.4). The genome of <em>Bacillus subtilis</em> is completely sequenced with well-established genetic and molecular modalities (BW.4). This microorganism is a commercially important producer of a diverse quantity of heterologous proteins, antibodies, vaccines, enzymes, and other secondary metabolites (BW.4). <em>B. subtilis</em> is not harmful to mammals as a species and is GRAS designated by FDA, meaning that this bacterium is benign to both humans and other mammals (BW.4). <em>B. subtilis </em>has less knowledge base relative to <em>E. coli</em> within the scope of iGEM, however its commercial relevance and safety made this microorganism an ideal candidate for this summer&rsquo;s experimentation.</p>
<h3>Experimental Design</h3>
<h4>Strain Selection</h4>
<p>Although <em>B. subtilis </em>is widely used as a production platform for synthesizing enzymes, fine chemicals, and pharmaceutical agents, this microorganism also has clear associated disadvantages (BW.5). <em>B. subtilis </em>secretes no fewer than seven proteases during vegetative growth and stationary phase, and contains three wall-associated proteases (BW.5). The two most abundant secreted proteases are the alkaline protease (AprE) and neutral protease (NprE) and are produced during early stationary phase (BW.5). Vegetative cells produce minor proteases and all three wall-associated ones (BW.5). Proteases, which are biochemical enzymes that break down proteins and peptides, potentially will impact CBCR expression and viability. In order to provide a control for this, the following strain selections were ordered for experimentation:</p>
<ol>
<li><em>B. subtilis KO7</em> (<em>BGSC Accession # 1A1134</em>): This strain was developed in June 2016 by the <em>Bacillus Genetic Stock Center</em>. This strain is a seven protease deletion strain which is free from antibiotic resistance genes or integrated plasmids. It was developed from the commonly used laboratory host PY79 by sequentially knocking out coded genes. All seven knockouts in KO7 were confirmed through sequencing to be marker-free (BW.5). If secreted proteases were responsible for CBCR degradation and loss of viability, this strain would theoretically work compared to a control (the host strain PY79).</li>
<li><em>B. subtilis PY79</em> (<em>BGSC Accession # 1A747</em>): This is a wildtype strain from which the KO7 strain was derived. This strain serves as a control when juxtaposed to KO7.</li>
<li><em>B. subtilis spo0A3</em> (<em>BGSC Accession # 1S1</em>): This strain comprises spo0A mutants, therefore they do not produce alkaline or neutral proteases. They also do not sporulate. The two most abundant secreted proteases are the alkaline protease (AprE) and the neutral protease (NprE) and are produced in the early stationary phase in normal <em>B. subtilis</em>. This mutant does not produce these proteases, however wall-associated proteases are still present. This strain is the result of mutations in the Spo0A locus and have normal antibiotic sensitivities. Note that Spo0A have drastically reduced genetic competence, so electroporation is recommended to transform them (BW.5).</li>
</ol>
<p>The <em>University of California - Santa Cruz </em>iGEM team also was working on a project implementing <em>B. subtilis </em>as an expression chassis. In order to promote collaboration, we shared the <em>B. subtilis spo0A3 </em>strain, the<em> B. subtilis PY70 </em>strain, and the <em>B. subtilis KO7 </em>strains with the UCSC iGEM team. These strains were obtained from Daniel Zeigler, the director of the Bacillus Genetic Stock Center. Expressed written permission was obtained from Dr. Zeigler on July 14th approving material transfer of these organisms between our teams.</p>
<h4>Genetic circuit</h4>
<p>Our genetic construction for <em>B. subtilis</em> was performed next. Our summer goal was to obtain expression of a single GAF domain in <em>B. subtilis</em>. Although operon structure manipulations and a host of CBCR attempts were performed in <em>E. coli</em>, a single expression success was the goal for <em>B. subtilis </em>provided the disparity of experimentation between <em>E. coli </em>and <em>B. subtilis</em> in iGEM.</p>
<p>We first considered ribosome-binding site (RBS) sites for <em>B. subtilis</em> and any impact such sequences had on expression in this chassis. It was found that changes in ribosome-binding sites of mRNA impacted translational efficiency in <em>B. subtilis </em>substantially (BW.6). In the work performed by this lab group, the influence of spacing between the Shine-Dalgarno sequence and the initiation codon was assessed (BW.6). <em>B. subtilis</em> translated messages with spacing shorter than optimal very inefficiently when compared to <em>E. coli </em>translation under analogous manipulations (BW.6). It was found that the degree of emphasis placed on the initiation codon type and spacing had a profound impact on translation (BW.6). It was found that the sequence AAA-AA<strong>T-AAG-GAG-G</strong>AA-AAA-AAA exhibited the highest specific activity (expressed in miller units) [BW.6]. Also documented is that in both <em>in vivo </em>and <em>in vitro</em> situations <em>B. subtilis </em>ribosomes required more extensive Shine-Dalgarno complementarity for efficient translation initiation and the effects of spacing were profound on expression (BW.6). With such considerations in mind, our RBS sequences were designed with the incorporation of this ideal spacing and Shine-Dalgarno for maximum expression activity.</p>
<p>The expression cascade for <em>B. subtilis</em> was designed as follows:</p>
<p>pVEG = A very strong constitutive promoter for <em>B. subtilis</em>. Chosen for its transcriptional strength and compatibility for <em>B. subtilis</em>. Total DNA insert size = 237 bp (BW. 7).</p>
<p>RBS A = Designed with two primer sequences annealed together, each containing segments for Golden Gate assembly with pVEG and HO. The annealed assembly is 21 bp long.</p>
<p>HO = The Heme Oxygenase genetic construct catalyzes the conversion of heme to Biliverdin IX alpha. This genetic construct is 726 bp (BW.8).</p>
<p>RBS B = Designed with two primer sequences annealed together, each containing segments for Golden Gate assembly with HO and PcyA. The annealed assembly is 21 bp long.</p>
<p>PcyA = Phycocyanobilin / ferredoxin oxidoreductase (PcyA) reduces Biliverdin IX alpha to PCB. This genetic construct is 750 bp (BW.8).</p>
<p>B0015 Terminator (used 2x) = iGEM part - double terminator consisting of BBa_B0010 and BBa_B012. This is the most commonly used iGEM terminator (BW.9). Biology: rrnBt1-T7TE, Chasis: <em>E. coli </em>and <em>B. subtilis</em>, Direction: Forward. Length: 129 bp.</p>
<p>IPTG Inducible Promoter = <a href="http://www.agscientific.com/iptg-isopropyl-beta-d-thiogalactoside.html">Isopropyl &beta;-D-1-thiogalactopyranoside</a>, abbreviated <a href="http://www.agscientific.com/iptg-isopropyl-beta-d-thiogalactoside.html">IPTG</a>, binds to the lac repressor and releases the tetrameric repressor from the lack operator in an allosteric manner, thus permitting lac operon gene transcriptase. Compatible in <em>E. coli </em>and in <em>B. subtilis</em>. We ordered this promoter through IDT as a g-block and cloned this gene out as well from <em>pHT43 </em>as two methods of derivation. Length: 107 bp</p>
<p>RBS C = Designed with two primer sequences annealed together, each containing segments for Golden Gate assembly with HO and PcyA. The annealed assembly is 21 bp long.</p>
<p>CBCR = Combines autocatalytically with phycocyanobilin to form a chromophore capable of absorbing and reflecting particular wavelengths, thus creating a pigmented protein. For this genetic setup, a NPF CBCR was selected. Length: 1451 bp</p>
<p>pHCMC04 = This is a 8,089 bp vector. It was chosen for its compatibility with <em>B. subtilis</em> and ability to promote high expression. This vector has proven stability advantages. This vector requires that Gram-positive compatible ribosome binding sites are selected. The internal PxylA promoter will transcribe all sites with the appropriate RBS. The vector coding sequence APE file from 1,203 - 2,692 was identified as a potential target site for the insertion of our DNA assembly.</p>
<p>Total Design Assembly bp: 3,499 bp - Total Vector with Assembly - 11,588bp</p>
<h4>Transformation Protocols</h4>
<p>A variety of <em>B.subtilis</em> competent preparation and transformation protocols were identified in order to obtain the optimal transformation efficiency. Manipulated strains, such as the spo0A3 mutant, are known to be difficult to transform unless electro-competent preparation and electro-transformation are performed (CITE). A variety of these protocols from literature were identified and are proposed below:</p>
<p>The first protocol was shared by the UC Santa Cruz with our team. The remainder of the protocols were identified by our team and shared with UCSC for their experimentation.</p>
<ol>
<li>Procedure 1: <em>Bacillus subtilis</em> - Multiporator / Eppendorf Eporator - Transformation Protocol No. 4308 915.504 - 08/2003 - Electrocompetent cells and Electroporation Transformation is described in this protocol. Protocol references the <em>Journal of Microbiological Methods </em>(1999).</li>
<li>Procedure 2: Transformation of <em>Bacillus subtilis </em>by Electroporation - <em>Institute of Horticultural Research, Littlehampton, West Sussex, </em>By Michele Stepheson and Paul Jarrett. Describes the use of electroporation performed using a Bio-Rad Gene Pulser and Pulse Controller as described by Dower et al. (1988). Illustrates PPEG as a superior electroporation medium than glycerol in the transformation of <em>B. subtilis </em>and that a particular ratio of transformation medium is required for maximum transformations. A tradeoff exists between the appropriate field strength and time constant in terms of transformation competence and bacterial cell mortality.</li>
<li>Procedure 3: High Transformation Efficiency of <em>B. Subtilis with integrative DNA Using Glycene Betaine as an Osmoprotectant</em> - <em>Fatma Meddeb-Mouelhi, Carlos Dulcey, and Marc Beauregaurd. </em>Theory: High electroporation efficiency (up to 5 x 10^5 cfu/microgram) was obtained using 7.5% glycene betaine. This 2011 protocol claims to substantially improve on the transformation efficiency of existing techniques. The electroporation technique relies on electric pulsation to induce a membrane potential for cells for competence and transformation.</li>
<li>Procedure 4: <em>A modified electro-transformation method for Bacillus subtilis and its application in the production of antimicrobial lipopeptides</em> - Theory: A modified electroporation method using trehalose is presented for the transformation of Bacillus subtilis. The new method improved the transformation efficiency of <em>B. subtilis</em> nearly 2,000 - fold compared with the usual method, giving 4 x 10^5 transformants/micro-gram DNA. Trehalose is a non-toxic sugar which protects biomolecules against environmental stress.</li>
<li>Procedure 5: <em>Groningen Method</em> - <em>Preparation of competent B. subtilis cells and transformation with plasmid DNA</em> - Theory: No electrocompetent preparation used for this protocol, solely starvation medium and minimal-growth medium solution.</li>
<li>Procedure 6: <em>Paris Method - Preparation of competent B. subtilis cells and transformation with plasmid DNA</em> - Theory: Specially formulated transformation and inoculation procedure to induce competent preparation without electroporation.</li>
</ol>
<p><br /><br /></p>
<h3 class = "inline">Results and Future Directions</h3><img src="pictures/UCD16_future.png" class = "img-rounded bactImg"alt="">
<p>With our experimental design in place our team began assembling the genetic construct and was able to complete it until the following structure:</p>
<p>However, due to PCR malfunctions, time constraints, and a issue in the core RBS sequence we were unable to proceed further with this portion of the project. In the future, our team would like to use complete our partial genetic construct. Then this segment could be integrated into the pHCMC04 vector and subsequent competent preparation and transformation protocols could thereby be enacted with the three strains (Wild type, protease knockout, wall associated knockout) to examine if successful holoCph1 protein (GAF domain expression) was possible.</p>
<p>If our future transformations and subsequent expression of the GAF protein in B.subtilis is successful, our goal is to express multiple GAF domains in this organism and quantify whether expression is comparable to that of E.coli.</p>
<p>If our future transformations and subsequent expression of the GAF protein in B.subtilis is unsuccessful, our goal is to first learn why and then try other GRAS organisms until we establish that GAF domains can be expressed in a GRAS organism and brought into the human food system.</p>
		</div>
	</div>
</div>