---
layout: page
title: Novel Expression
---
<div class="container-fluid">
	<div class="row row-centered body_odd">
		<div class="col-centered col-xs-10 ">
			<h1>Novel Expression</h1>
			<p>As a result of our search for homologs to blue GAF proteins, our team selected 13 GAF domain to transform into E.coli. We were able to successfully transform 4 of them, 3 of which were blue and 1 red. </p>
		</div>
	</div>
</div>
<div class="container">
	<div class="row row-centered">
		<div class="col-centered col-xs-12">
			<h3>Motivation</h3>
			<p>A main focus of our project was to engineer a replacement for the dye Blue #1. While there are some known GAF proteins which absorb only 10 nm away from Blue #1, this actually represents large visual differences in color (25). For example, the difference in color from Spirulina to Blue #1 is only 10nm, and that is clearly visible by eye (26). In an effort to match Blue #1’s peak spectra better, our team chose 13 blue GAF domain homologs in an effort to expand the known blue spectra of GAF domains, and ultimately get closer to ideal absorption of 630nm (8). </p>
			<h3>Research</h3>
			<p>After carefully selecting our 13 GAF proteins we began their transformation into E.coli. For easy induction, the DNA for the coding sequences were synthesized with a pBAD promoter (K206000) and strong RBS (B0034) so cultures can be induced with arabinose to produce the apoprotein.</p>
			<p>Due to cloning difficulties, we resorted to using standard digestion/ligation methods to insert the protein-coding sequences into pSB1C3 sourced from J04450. Correctly transformed cells were screened by selecting white colonies over red colonies. These were sequenced and co-transformed into DH5-alpha cells along with a plasmid containing the PCB pathway enzymes. The co-transformed cells were screened using a double-antibiotic plate, and a correctly co-transformed colony was used to innoculate 50ml TB cultures induced with arabinose. These cultures were harvested after overnight shaking for use in protein purification.</p>
			<p>After cell lysis by sonication, the proteins were further purified using the intein mediated purification with an affinity chitin-binding tag (21). This is a purification method utilizes the inducible self-cleaving inteins to separate the protein from the affinity tag without the use of the protease (21). This purification method was chosen because it allows complete removal of the intein tag from the protein, meaning no foreign DNA is remaining on the protein after purification. Our team felt this is an important step in relation to the labeling scheme our dye may undergo in which having even a piece of non-native DNA could mean dramatic differences. A more thorough discussion of the labeling possibilities can be found here (link to HP stuff).</p>
			<p>Our team discovered the purification of these proteins ultimately had a noticeable effect on the final absorption spectra. The main difference being the purified protein has a much clearer and refine absorbance spectra and does not have other residues around the 400 nm range. However, one downside that is that the purification does lead to a loss in product, as evidenced by the reduction in peak height. This loss of product can ultimately be improved in the future more efficient eluting methods.</p>
			<div class="centered"><img src="pictures/UCD16_novel1.png" alt=""></div>
			<p>After purification the spectra of all the novel GAF domains and Blue #1 were examined for comparison. _____ conclusions based on what happens.</p>
			<div class="centered"><img src="pictures/UCD16_novel2.png" alt=""></div>
			<h3 class = "inline">Future Directions</h3><img src="pictures/UCD16_future.png" class = "bactImg img-rounded" alt="">
			<p>In the future, our team would like to work towards expanding the GAF domain spectra by both continuing the search for GAF homologs and also by mutations. By continuing the search for GAF homologs we feel the spectra can be significantly more filled; however, given that small changes in peak absorption visually have a significant difference in color,  we also think forcing mutations in known GAF domains is the next step. Similar to mutations performed in Roger Y. Tsien’s lab to which established that green fluorescent protein (GFP) spectral characteristics could be dramatically changed by single point mutations, our team hypothesizes that this may be the case for GAF domains as well (22). There is preliminary evidence that shows _____ () meaning that it is possible that GAF proteins spectral characteristics could be influenced by preformed intentional mutations in protein sequences. </p>
		</div>
			<div class="col-xs-6 col-centered projLinks">
				<a href="{{"opt.html"| prepend: site.baseurl }}" class="navBtn">Next: Production Optimization</a>
			</div>	
<br><hr>
	<div class="row row-centered footNotes">
		<div class="col-xs-12 col-centered">
<ol>
<li>&nbsp;<a href="http://www.chicagotribune.com/business/ct-artificial-food-coloring-0626-biz-20160624-story.html">http://www.chicagotribune.com/business/ct-artificial-food-coloring-0626-biz-20160624-story.html</a></li>
<li><a href="http://sensing.konicaminolta.us/products/">http://sensing.konicaminolta.us/products/</a></li>
<li><a href="http://www.mars.com/global/about-us/policies-and-practices/color-policy">http://www.mars.com/global/about-us/policies-and-practices/color-policy</a></li>
<li><a href="http://www.nbcnews.com/business/consumer/kraft-mac-cheese-says-goodbye-dye-n345006">http://www.nbcnews.com/business/consumer/kraft-mac-cheese-says-goodbye-dye-n345006</a></li>
<li><a href="http://blog.generalmills.com/2015/06/a-big-commitment-for-big-g-cereal/">http://blog.generalmills.com/2015/06/a-big-commitment-for-big-g-cereal/</a></li>
<li><a href="http://www.nestleusa.com/media/pressreleases/nestl%C3%A9-usa-commits-to-removing-artificial-flavors-and-fda-certified-colors-from-all-nestl%C3%A9-chocolate-candy-by-the-end-of-20">http://www.nestleusa.com/media/pressreleases/nestl%C3%A9-usa-commits-to-removing-artificial-flavors-and-fda-certified-colors-from-all-nestl%C3%A9-chocolate-candy-by-the-end-of-20</a></li>
<li><a href="http://www.fao.org/docrep/016/ap106e/ap106e.pdf">http://www.fao.org/docrep/016/ap106e/ap106e.pdf</a></li>
<li><a href="https://pubchem.ncbi.nlm.nih.gov/compound/Acid_Blue_9#section=Stability">https://pubchem.ncbi.nlm.nih.gov/compound/Acid_Blue_9#section=Stability</a></li>
<li><a href="http://www.fda.gov/ForIndustry/ColorAdditives/ColorAdditiveInventories/ucm115641.htm">http://www.fda.gov/ForIndustry/ColorAdditives/ColorAdditiveInventories/ucm115641.htm</a></li>
<li><a href="http://link.springer.com/article/10.1007/s00217-004-1062-7"> http://link.springer.com/article/10.1007/s00217-004-1062-7</a></li>
<li><a href="http://www.instructables.com/id/Blue-Foods-Colorful-cooking-without-artificial-dy/"> http://www.instructables.com/id/Blue-Foods-Colorful-cooking-without-artificial-dy/</a></li>
<li>A Brief History of Phytochrome Nathan C. Rockwell and J. Clark Lagarias</li>
<li>Hirose, Yuu, et al. "Green/red cyanobacteriochromes regulate complementary chromatic acclimation via a protochromic photocycle."<em>Proceedings of the National Academy of Sciences</em> 110.13 (2013): 4974-4979.</li>
<li>Rockwell, Nathan C., and J. Clark Lagarias. "A brief history of phytochromes." <em>ChemPhysChem</em> 11.6 (2010): 1172-1180.</li>
<li>Bussell, Adam N., and David M. Kehoe. &ldquo;Control of a Four-Color Sensing Photoreceptor by a Two-Color Sensing Photoreceptor Reveals Complex Light Regulation in Cyanobacteria.&rdquo; <em>Proceedings of the National Academy of Sciences of the United States of America</em> 110.31 (2013): 12834&ndash;12839. <em>PMC</em>. Web. 7 Oct. 2016.</li>
<li><a href="http://www.nutritionaloutlook.com/food-beverage/coloring-spirulina-blue"> http://www.nutritionaloutlook.com/food-beverage/coloring-spirulina-blue</a></li>
<li>Nature&rsquo;s Palette: The Search for Natural Blue Colorants</li>
<li>https://www.novozymes.com/en</li>
<li><a href="http://www.dsm.com/corporate/home.html"> http://www.dsm.com/corporate/home.html</a></li>
<li><a href="http://prodata.swmed.edu/promals3d/promals3d.php"> http://prodata.swmed.edu/promals3d/promals3d.php</a></li>
<li><a href="https://www.neb.com/products/e6901-impact-kit"> https://www.neb.com/products/e6901-impact-kit</a></li>
<li>Heim R, Cubitt A, Tsien R (1995).<a href="http://tsienlab.ucsd.edu/Publications/Heim%201995%20Nature%20-%20Improved%20GFP.PDF"> "Improved green fluorescence"</a> (PDF). <em>Nature</em>. 373 (6516): 663&ndash;4.<a href="https://en.wikipedia.org/wiki/Digital_object_identifier">doi</a>:<a href="https://dx.doi.org/10.1038%2F373663b0">10.1038/373663b0</a>.<a href="https://en.wikipedia.org/wiki/PubMed_Identifier"> PMID</a><a href="https://www.ncbi.nlm.nih.gov/pubmed/7854443"> 7854443</a>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/genome/browse/"> https://www.ncbi.nlm.nih.gov/genome/browse/</a></li>
<li><a href="http://pfam.xfam.org/"> http://pfam.xfam.org/</a></li>
<li>Red/Green Cyanobacteriochromes: Sensors of Color and Power - lagarias</li>
<li><a href="https://en.wikipedia.org/wiki/Brilliant_Blue_FCF#/media/File:Blue_smarties.JPG"> https://en.wikipedia.org/wiki/Brilliant_Blue_FCF#/media/File:Blue_smarties.JPG</a></li>
<li>Harnessing phytochrome's glowing potential - clark</li>
<li><a href="http://www.sciencedirect.com/science/article/pii/S1359511312000268"> Stability of phycocyanin extracted from Spirulina sp.: Influence of temperature, pH and preservatives</a>- R Chaiklahan, N Chirasuwan, B Bunnag - Process Biochemistry, 2012 - Elsevier</li>
<li>http://www.lightboxkit.com/Assay_dye.html</li>
<li>"Your Brain - A User's Guide - 100 Things You Never Knew." National Geographic Time INC. Specials 12 Feb. 2016: Print.</li>
<li>Sensing, Konica Minolta. "How Color Affects Your Perception of Food." <em>Konica Minolta Color, Light, and Display Measuring Instruments</em>. Konica Minolta Sensing Americans Inc., 2006. Web. 10 Oct. 2016.</li>
<li>"Center for Disease Control and Prevention - General Information Escherichia Coli." <em>CDC 24/7: Saving Lives, Protecting People</em>. Centers for Disease Control and Prevention, 06 Nov. 2015. Web. 10 Oct. 2016.</li>
<li>Olmos J, Paniagua-Michel J (2014) Bacillus subtilis A Potential Probiotic Bacterium to Formulate Functional Feeds for Aquaculture. J Microb Biochem Technol 6: 361-365. doi:10.4172/1948-5948.1000169</li>
<li>Zeigler, Daniel R. "Bacillus Genetic Stock Center." <em>Bacillus Genetic Stock Center Website</em>. The Bacillus Genetic Stock Center - Biological Sciences, Oct. 2016. Web. 10 Oct. 2016.</li>
<li>Vellanoweth, Robert Luis, and Jesse C. Rabinowitz. "The Influence of Ribosome binding-sites Elements on Translational Efficiency in Bacillus Subtilis and Escherichia Coli in Vivo." Molecular Microbiology 1105-1114, 15 Jan. 1992. Web. 9 Oct. 2016.</li>
<li>"Pveg (Plasmid #55173)." <em>Addgene - The Nonprofit Plasmid Repository</em>. Synthetic Biology; Bacillus BioBrick Box, Aug. 2016. Web. 10 Oct. 2016.</li>
<li>Gambetta, Gregory A., and Clark J. Lagarias. "Genetic Engineering of Phytochrome Biosynthesis in Bacteria." <em>Section: Molecular and Cellular Biology</em>. Proceedings of the National Academy of Sciences of the United States of America, 19 July 2001. Web. 9 Oct. 2016.</li>
<li>"Registry of Standard Biological Parts - B0015." <em>Part: BBa_B0015</em>. International Genetically Engineered Machine Competition (iGEM), 17 July 2003. Web. 9 Oct. 2016.</li>
<li>"Bacillus Subtilis - Indiamart." IndiaMART InterMESH Ltd., n.d. Web. 9 Oct. 2016.</li>
</ol>

		</div>
	</div>		
	</div>
</div>